closed/src/java.base/share/classes/com/sun/crypto/provider/NativeGaloisCounterMode.java

```diff
@@ -467,13 +467,12 @@ int decrypt(byte[] in, int inOfs, int len, byte[] out, int outOfs) {
      */
     int decryptFinal(byte[] in, int inOfs, int len,
                      byte[] out, int outOfs)
-        throws IllegalBlockSizeException, AEADBadTagException,
-        ShortBufferException {
+        throws IllegalBlockSizeException, ShortBufferException {
         if (len < 0) {
             throw new ProviderException("Input length is negative");
         }
         if (len < tagLenBytes - ibuffer.size()) {
-            throw new AEADBadTagException("Input too short - need tag");
+           throw new IllegalBlockSizeException("Input too short - need tag");
```

Comment:
Please don't change the indentation (here or line 508); it was correct before.

Reply:
Added back a space.